core/src/main/java/com/scalar/db/transaction/consensuscommit/TwoPhaseConsensusCommit.java

```diff
@@ -164,12 +159,10 @@ public void rollback() throws RollbackException {
       return;
     }
 
-    if (isCoordinator) {
-      try {
-        commit.abortState(crud.getSnapshot().getId());
-      } catch (UnknownTransactionStatusException e) {
-        throw new RollbackException("rollback failed", e);
-      }
+    try {
+      commit.abortState(crud.getSnapshot().getId());
```

Comment:
Ditto. Regardless that it's a coordinator or a participant, we always call this `commit. abortState()` here.